app/carbonapi/app.go

```diff
@@ -42,6 +42,7 @@ import (
 // BuildVersion is provided to be overridden at build time. Eg. go build -ldflags -X 'main.BuildVersion=...'
 var BuildVersion string
```

Comment:
We have public BuildVersion in package carbonapi as well. Why we need duplicates?

Reply:
Good point. We will look into it in the next refactoring step.